boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -95,6 +95,15 @@
 		};
 	};
 
+	nrf_radio_coex: nrf7002-coex {
+		status = "okay";
+		compatible = "nordic,nrf700x-coex";
+		req-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		pri-dir-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
+		grant-gpios = <&gpio0 24 GPIO_ACTIVE_HIGH>;
+		swctrl1-gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
```

Comment:
These pins should be also added in the `nrf-gpio-forwarder` node so that they are assigned to the NET core.

Reply:
They are assigned to the NET core without using the `nrf-gpio-forwarder` - take a look here: https://github.com/ahasztag/sdk-nrf/blob/c3d4551488ba56f5b7356d4ef8a62517e63b6fc5/subsys/mpsl/cx/nrf700x/mpsl_cx_nrf700x.c#L283. Introducing the pin forwarder is not needed additionaly. 

We discussed with @jciupis if instead of passing the pins to the NET core through code the pin forwarder could be used on nRF53. However, there is one big problem with it - the way it is now, the same shield overlay, `nrf7002_ek.overlay` is needed for the nRF53 application core and nRF52 (only an additional overlay for the network core is used). If we were supposed to use the pin forwarder, we would have to use a different DTS overlay for the EK for nRF53 and nRF52 - it seems it is not the way the EK overlays are supposed to be created, there should be one overlay for different SoCs.

So the pin forwarder in this case seems to create many problems and not really helping with anything.

Reply:
> So the pin forwarder in this case seems to create many problems and not really helping with anything.

It helps by allowing us to remove code associated with `CONFIG_MPSL_CX_PIN_FORWARDER`. But it does also create a problem with the EK overlay that I have no clean idea of solving. My best shot was here: https://github.com/nrfconnect/sdk-nrf/pull/8750#discussion_r972877276, but it has its own drawbacks too.